packages/oauth/src/index.ts

```diff
@@ -9,10 +10,31 @@ import {
   GenerateInstallUrlError,
   AuthorizationError,
 } from './errors';
-import { parse as parseUrl, URLSearchParams, URL } from 'url';
 import { Logger, LogLevel, getLogger } from './logger';
 import { MemoryInstallationStore } from './stores';
 
+// default implementation of StateStore
```

Comment:
Out of curiosity, what's the reasoning behind moving this up? Was this related to one of the eslint rules?

Reply:
Yes, the 'no use before defined' rule. While functions are hoisted to the top by the runtime, classes, and I guess types as well, are not.